src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -227,6 +235,10 @@ public HttpResponse doAbort() {
         preAbortCheck();
 
         FlowInterruptedException e = new FlowInterruptedException(Result.ABORTED, new Rejection(User.current()));
+
+        // notify this input action was abort
+        notifyInput(InputExtension.NotifyEvent.ABORT);
```

Comment:
Execute before return ? in case some exceptions

Reply:
IMO, it should never occur an exception at here.